src/net.h

```diff
@@ -1,5 +1,6 @@
 // Copyright (c) 2009-2010 Satoshi Nakamoto
 // Copyright (c) 2009-2016 The Bitcoin Core developers
+// Copyright (c) 2021 The Dogecoin Core developers
```

Comment:
seems to be no other change than the copyright message?

Reply:
That's right; I was over-eager applying a macro to all of my open files. Removed.